kube/app/deployment.yml

```diff
@@ -108,13 +108,13 @@ spec:
                   key: session-secret
             - name: FILE_VAULT_URL
             {{ if eq .KUBE_NAMESPACE .PROD_ENV }}
-              value: https://fv.prod.{{ .APP_NAME }}.homeoffice.gov.uk/file
+              value: https://fv-{{ .APP_NAME }}.sas.homeoffice.gov.uk/file
             {{ else if eq .KUBE_NAMESPACE .STG_ENV }}
-              value: https://fv-stg.prod.{{ .APP_NAME }}.homeoffice.gov.uk/file
+              value: https://fv-{{ .APP_NAME }}.stg.sas.homeoffice.gov.uk/file
```

Comment:
As there is no staging environment for this project, perhaps it should be removed.